CMakeLists.txt

```diff
@@ -44,6 +47,24 @@ set(SEALAPI_SOURCES
 pybind11_add_module(_sealapi_cpp ${SEALAPI_SOURCES})
 
 if(NOT WIN32)
+    target_link_libraries(libtenseal PRIVATE seal)
```

Comment:
This also could be conditioned by an environment variable BUILD_TEST

Reply:
The same as above